src/authenticators/peer_credentials_authenticator/mod.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Contributors to the Parsec project.
```

Comment:
Should we commit that file in this PR? Are you trying to sneakily make it in 👀 ?

Reply:
Haha, oops. Removed. 😄